docs/content/installation/csi_ug_requirements.md

```diff
@@ -0,0 +1,69 @@
+# Compatibility and requirements
+
+For the complete and up-to-date information about the compatibility and requirements for using the IBM® block storage CSI driver, refer to its latest release notes. The release notes detail supported operating system and container platform versions, as well as microcode versions of the supported storage systems.
+
+Before beginning the installation of the CSI (Container Storage Interface) driver, be sure to verify that you comply with the following prerequisites.
+
+For IBM Cloud® Satellite users, see [cloud.ibm.com/docs/satellite](https://cloud.ibm.com/docs/satellite) for full system requirements.
+
+**Important:** When using Satellite, complete the following checks, configurations, and the installation process before assigning the hosts to your locations. </br>In addition, **do not** create a Kubernetes cluster. This is done through Satellite.
+
+-   The CSI driver requires the following ports to be opened on the worker nodes OS firewall:
+    -   **For all iSCSI users**
+
+        Port 3260
+
+    -   **FlashSystem A9000 and A9000R**
+
+        Port 7778
+
+    -   **IBM Spectrum® Virtualize Family includes IBM® SAN Volume Controller and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
+
+        Port 22
+
+    -   **DS8000® Family systems**
+
+        Port 8452
+
+-   Be sure that multipathing is installed and running.
+
+Perform these steps for each worker node in Kubernetes cluster to prepare your environment for installing the CSI (Container Storage Interface) driver.
+
+1. **For RHEL OS users:** Ensure iSCSI connectivity. If using RHCOS or if the packages are already installed, skip this step and continue to step 2.
```

Comment:
@rivkap in the original [procedure](https://www.ibm.com/docs/en/stg-block-csi-driver/1.4.0?topic=installation-compatibility-requirements) (step 1), the names of the packages (`iscsi-initiator-utils` and `xfsprogs`) with their installation commands were explicitly specified, but since this change we no longer have them.
could we restore them?

Reply:
@oriyarde  - as I mentioned on Slack - this is a part that we decided was "advanced"
@barakda  @ArbelNathan   - is this something that we want to put back in?

Reply:
I remember this decision was related to ssh being not recommended, but I think we should at least keep the names of the packages

Reply:
I think that we need at least the package's name.

Reply:
@oriyarde  @ArbelNathan please re-review and let me know what to put in the latest pull request - https://github.com/IBM/ibm-block-csi-driver/pull/360

Reply:
@rivkap please see latest list in CSI-3397